apps/sharing/fixtures/sharing/share_counts.json

```diff
@@ -4,7 +4,7 @@
         "model": "stats.addonsharecounttotal",
         "fields": {
             "count": 13,
-            "service": "delicious",
+            "service": "facebook",
             "addon": 3615
         }
     },
```

Comment:
You can completely remove this, from line 2 to 10 included. We need at least one sharing service with no shares for the test.